plugin-build/src/main/kotlin/io/sentry/android/gradle/instrumentation/AbstractSpanAddingMethodVisitor.kt

```diff
@@ -66,7 +66,7 @@ abstract class AbstractSpanAddingMethodVisitor(
         visitVarInsn(Opcodes.ALOAD, spanIndex) // span
         visitJumpInsn(Opcodes.IFNULL, gotoIfNull)
         visitVarInsn(Opcodes.ALOAD, spanIndex) // span
-        visitLdcInsn("db")
```

Comment:
Or maybe it will be better if we actually keep `db.query` for room spans, but for sqlite ones change it to `db.sql.query` as per documentation?

Reply:
Sounds to me that `db.query.mapping` sounds more correct than `db.query` but then having it as a child/parent thing is odd. 

Do we know the exact time of the mapping so we can have:

_ `db`
|_ `db.orm.mapping`
|_ `db.query`

Though I just threw in even more options.
Maybe @marandaneto @AbhiPrasad or @rhcarvalho have better insight here.

Reply:
We could know the time of the mapping, but this would mean re-implementing everything we did with the bytecode, so maybe not now and also I don't see this straightforward

Reply:
I think the room span can also just be `db` if we are going to work toward adding the mapping timing at a later point.

Tbh, I'm fine with just shipping in it's existing state and just iterating based on customer feedback - if someone feels strongly we can re-evaluate to improve accordingly.

Reply:
IMO it'd make more sense either to introduce a `db.orm` as an action or just use plain `db` for the room operation.
but for the SQLite span, I'd rather use `db.sql.query` because it's exactly what's been doing.
wdyt?

Reply:
Yeah, sounds good to me, I will use plain `db` for room

Reply:
Changed it to this 
![image](https://user-images.githubusercontent.com/4999776/143246692-e656843b-55a5-43cb-baca-d65bf4503f9c.png)

Reply:
👍